app/src/main/java/com/asfoundation/wallet/topup/TopUpFragment.kt

```diff
@@ -142,6 +146,7 @@ class TopUpFragment : DaggerFragment(), TopUpFragmentView {
       selectedCurrency = savedInstanceState.getString(SELECTED_CURRENCY_PARAM) ?: FIAT_CURRENCY
       localCurrency = savedInstanceState.getSerializable(LOCAL_CURRENCY_PARAM) as LocalCurrency
     }
+    savedInstanceState?.run { selectedPaymentMethod = getInt(SELECTED_PAYMENT_METHOD_PARAM) }
```

Comment:
I know both work, but use let instead since its been the pattern used for nulls. 
Don't you think it's better to use a default value 0 to avoid a NPE on the possibility of having savedInstance but no value saved yet?

Reply:
I choose run because let gets more verbose in this case. But I'll change it.
selectedPaymentMethod isn't nullable, it's already initialised with the value 0 and getInt will return 0 by default anyway.